src/index.scss

```diff
@@ -377,7 +378,7 @@ PACKAGES
 	padding-bottom: 15px;
 	.react-fine-uploader-gallery-dropzone, .react-fine-uploader-gallery-nodrop-container {
 		min-height: 120px;
-		border: 1px dashed darkgrey;
+		//border: 1px dashed darkgrey;
```

Comment:
Should clean this up, if it's not needed, just delete it. If we need it again, it's always in the git history